openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -141,6 +141,31 @@ <h2>{% trans 'Interface status' %}: {{ interface.name }}</h2>
                   {% endif %}
                   {% endif %}
               </div>
+              <div class = "device_addresses">
+                {% if interface.addresses %}
+                  <h2>{% trans 'ADDRESSES' %}:</h2>
```

Comment:
there's no need to write it uppercase, that's CSS's job.

The fact that the table contain IP addresses is self explanatory so this heading is not needed. You can remove it.